devel/gcli/Portfile

```
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem            1.0
+
+PortGroup             github 1.0
+
+name                  gcli
+version               2.3.0
+revision              0
+
+categories            devel
+
+license               BSD
+
+platforms             darwin
+maintainers           nomaintainer
+
+description           CLI utility to interact with various Git forges
+
+long_description      GCLI is a simple and portable CLI tool for interacting with GitHub, \
+                      GitLab, and Gitea from the command line.
+
+homepage              https://herrhotzenplotz.de/gcli/
+
+github.setup          herrhotzenplotz ${name} ${version} v
+github.tarball_from   archive
+
+checksums             rmd160  e83b12f4f618f81ae79ea7e2a096b0227b64fedd \
+                      sha256  3e25d9667ed6317087f2425390f4f4f1890b21bbaa7652c1e03fe34038e24229 \
+                      size    391000
+
+depends_lib           port:curl
+
+depends_build         port:bison \
```

It's a best practice to use `-append` suffixes for the `depends_*` lists.  Certain facilities such as portgroups may automatically add entries to `depends_lib` or `depends_build` lists.  So using `depends_lib-append` and `depends_build-append` will ensure nothing gets overwritten.

```suggestion
depends_build-append  port:bison \
```